### ISSUE TITLE
launch.json envFile not escaped before injecting

### ISSUE THREAD

**Issue by itslenny** [Microsoft/vscode#61902]
- VSCode Version: Version 1.28.2 (1.28.2)
- OS Version: OSX 10.14

Steps to Reproduce:

1. add `"envFile": "${workspaceRoot}/.env"` to `launch.json`
2. add value to `.env` that contains `!` characters

```bash
my_secret=SomeAwesomeSecret!!!
```

Does this issue occur when all extensions are disabled?: Yes

Because `!` is previous command in bash when it loads the `.env` file it fails catastrophically. These characters (and probably others) should be escaped when `.env` is loaded or perhaps it should be loaded in a way that doesn't try to execute things like that.

**Comment by weinand** [comment]
I don't think that this is related to the use of "envFile".
Most likely the same happens when using
```
"env": {
   "my_secret": "SomeAwesomeSecret!!!"
}
```

**Comment by lheneks** [comment]
I just want to add that the pipe character | will also cause failures

**Comment by weinand** [comment]
Fixed with PR #68435.  Thanks @geirha.
